### DISCUSSION TITLE
Tutorial ISMB/ECCB 2023

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/atlas#677] [General]
https://silask.github.io/talk/tutorial-at-the-ismb-2023/

**Comment by SilasK** [comment]
Hello, my name is Silas, I am the main developper of metagenome metagenome-atlas.
I currently work at Nestlé Research on human microbiomes..

**Comment by lewhiteside** [comment]
Hello, my name is Louise and I am a PhD candidate at Queens University Belfast.  My research is focused on eDNA meta-anlyses within microbial surveillance strategies - specifically looking at the wastewater microbiome in public health surveillance across Northern Ireland.

**Comment by greicianepaneto** [comment]
Hello, my name is Greiciane. I am a associate professor at Federal University of Espirito Santo, Brazil. I am not working with metagenomics yet, but I would like to. I work with NGS of citrus plants (using DArT-Seq) and SARS-COV-2 (using Nanopores).

**Comment by nicholascdove** [comment]
Hi all, I'm Nicholas Dove, and I am a computational biologist at [AgBiome](https://agbiome.com/). I work mostly with environmental (soil/rhizosphere/plant-associated) genomes and metagenomes.

**Comment by mars188** [comment]
Hello, I am Arshad working as a bioinformatics data analyst at New York University Abu Dhabi [https://nyuad.nyu.edu/en/research/faculty-labs-and-projects/nyuad-cgsb/cgsb-cores/bioinformatics-core/muhammad-arshad.html](url), and working on human as well as environmental microbiome with shotgun and 16s data.

**Comment by SilasK** [comment]
@sinamajidian Thank you for noticing that the link to the unix tutorial was broken. [Here is the correct one](https://edu.sib.swiss/course/view.php?id=82)

**Comment by tinavisnovska** [comment]
Hi, I am Tina, a bioinformatician based in a norwegian university hospital near Oslo (ahus university hospital). Trying now to fish metatranscriptomic data out of human RNAseq and to make sense of the data. Thus interested in functional analysis of low volume samples and all the related topics.
